src/lib/Sympa/HTMLDecorator.pm

```diff
@@ -247,13 +246,17 @@ sub decorate_email_gecos {
             } else {
                 $decorated .= $item->{text};
             }
+        } elsif ($item->{event} eq 'start'
+            and $item->{attr}
+            and 0 == index(lc($item->{attr}->{href} // ''), 'mailto:')) {
+            my $text = $item->{text};
+            $text =~ s{(?<=\bhref=)[^\s>]+}{"mailto:"}gi;
```

Comment:
Can you explain me the regex? I don’t get it :thinking:

Reply:
`(?<=`...`)` is looking-behind assertion.  Matching will succeed if the pattern that assertion includes precedes (See [tutorial](https://perldoc.perl.org/perlretut#Looking-ahead-and-looking-behind) for details). 

`(?<=\bhref=)[^\s>]+` may be read as "a string matching with `[^\s>]+` preceded by a string matching with `\bhref=`".  Thus, in a string
> `<a href="mailto:me@example.org">`

only `"mailto:me@example.org"` matches: Assertion is not included in the range of match.

Therefore, this line replaces the value of `href` attribute in the tag with a empty mailto URL "`mailto:`".

Reply:
Ok, thanks. I never used looking-behind assertion, so didn’t knew the syntax.

It could be useful for future people like me to add a comment like `# Emptying mailto link target`, don’t you think? That’s just a suggestion, not mandatory.

Reply:
> It could be useful for future people like me to add a comment like `# Emptying mailto link target`, don’t you think? That’s just a suggestion, not mandatory.

Makes sense.  I’ll do it.  Thanks for advice!